spec/jobs/send_webhook_job_spec.rb

```diff
@@ -34,7 +36,7 @@
     end
 
     it 'calls the webhook invoice service' do
-      described_class.perform_now(:add_on, invoice)
+      subject.perform_now(:add_on, invoice)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)